x-pack/platform/plugins/shared/onechat/common/http_api/tools.ts

```diff
@@ -50,3 +50,7 @@ export type BulkDeleteToolResult = BulkDeleteToolSuccessResult | BulkDeleteToolF
 export interface BulkDeleteToolResponse {
   results: BulkDeleteToolResult[];
 }
+
+export interface ExecuteToolResponse {
+  result: any;
```

Comment:
With structured tool calls I believe that tool response should have known return type, e.g. esql tools return tabular data?

See https://github.com/elastic/kibana/pull/229562